src/utils.ts

```diff
@@ -168,3 +168,47 @@ export function getStyleForOverlappingEvent(
   }
   return overlapStyle
 }
+
+export function getDatesInNextCustomDays(
+  date: Date | dayjs.Dayjs = new Date(),
+  weekStartsOn: WeekNum = 0,
+  weekEndsOn: WeekNum = 6,
+  locale = 'en',
+) {
+  const subject = dayjs(date)
+  const subjectDOW = subject.day()
+  const days = Array(weekDaysCount(weekStartsOn,weekEndsOn))
+    .fill(0)
+    .map((_, i) => {
+      return subject.add(i - subjectDOW + weekStartsOn, 'day').locale(locale)
+    })
+  return days
+}
```

Comment:
Can you write a unit test for this? I think it's easier to write unit tests for these kind of code.

Reply:
I have to look into it, i will do it asap.

Reply:
Thanks!